nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -111,7 +71,48 @@ type Props = {
   imsi: string,
 };
 
-function JsonDialog(props: Props) {
+export function SubscribersOverview() {
+  const {relativePath, relativeUrl} = useRouter();
+  const networkCtx = useContext(NetworkContext);
+  const tabs = [
+    {
+      label: 'Sessions',
+      to: '/sessions',
+      icon: PeopleIcon,
+    },
+  ];
+
+  return (
+    <>
+      <TopBar
+        header={TITLE}
+        tabs={
+          networkCtx.networkType !== LTE
```

Comment:
I again check this one. Looks like we may have cases where we will use FEG_LTE networks with local subs. We might have to display both the tabs.